creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -409,17 +409,48 @@ spec:
   - name: red
     multus:
       networkName: red
------
+----
 
 At this time, `bridge` mode doesn’t support additional configuration
 fields.
 
 ______________________________________________________________________________
-*Note:* due to IPv4 address delagation, in `bridge` mode the pod doesn’t have
+*Note:* due to IPv4 address delegation, in `bridge` mode the pod doesn’t have
 an IP address configured, which may introduce issues with third-party solutions
 that may rely on it. For example, Istio may not work in this mode.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* there is an option to disable bridge interface with pod network.
```

Comment:
I am sorry for not catching it yesterday (in my defense, it was too late and I was sleepy already) but: this first sentence is not very clear. Specifically, one could read it as: "There is an option that allows a user/admin to disable adding a pod interface to a VMI (that is usually/by default of bridge type)." I think it would be great if we can reword it to remove this ambiguity. For example,

```
Note: Admin can forbid using `bridge` interface type for pod networks via a designated configuration flag. To achieve it, the admin should set the following option to `true`:
...
```